src/docs/self-hosted/troubleshooting.mdx

```diff
@@ -97,6 +97,23 @@ worker1:
 
 To see a more complete example, please see [a sample solution on our community forum](https://forum.sentry.io/t/how-to-clear-backlog-and-monitor-it/10715/14?u=byk).
 
+## Postgres
+
+Postgres is used for the primary datastore, as well as the [nodestore](https://develop.sentry.dev/services/nodestore/) which is used to store key/value data. The `node_nodestore` table can grow rapidly, especially when heavily utilising the Performance Monitoring feature as trace data is stored in this table.
+
+The `node_nodestore` table is cleaned up as part of the `cleanup` task, however Postgres may not get a chance to vacuum the table (especially under heavy load), so even the rows may be deleted, they're still taking up space on disk.
+
+You can use pg-repack which repacks a table live by creating a new table and copying data across, before dropping the old one. You'll want to run this after the clean up script, and note that as it creates a table, disk usage will spike before going back down.
+
+An example script below:
+
+```
+# Only keep the last 7 days of nodestore data. We heavily use performance monitoring.
+docker-compose exec -T sentry-cleanup sentry cleanup --days 7 -m nodestore -l debug
+# This ensures pg-repack exists before running as the container gets recreated on upgrades
+docker-compose exec -T postgres bash -c "apt update && apt install -y postgresql-9.6-repack && su postgres -c 'pg_repack -E info -t nodestore_node'"
+```
```

Comment:
```suggestion
```shell
# Only keep the last 7 days of nodestore data. We heavily use performance monitoring.
docker-compose run -T web cleanup --days 7 -m nodestore -l debug
# This ensures pg-repack exists before running as the container gets recreated on upgrades
docker-compose run -T postgres bash -c "apt update && apt install -y --no-install-recommends postgresql-9.6-repack && su postgres -c 'pg_repack -E info -t nodestore_node'"
```
```